grocery-list.json

```diff
@@ -9,8 +9,8 @@
       "quantity": "one dozen"
     },
     {
-      "item": "Peanuts",
-      "quantity": "1 ton"
+      "item": "cookies",
+      "quantity": "all of them"
```

Comment:
But if we don't get 1 ton of peanuts, what will we feed the elephants? :peanuts: